README.md

```diff
@@ -1,96 +1,75 @@
-| Marina Moore          | `ec81669734e017996c5b85f3d02c3de1dd4637a152019fe1af125d2f9368b95e` (v9+) `25a0eb450fd3ee2bd79218c963dce3f1cc6118badf251bf149f0bd07d5cabe99` (v5-8) `eaf22372f417dd618a46f6c627dbc276e9fd30a004fc94f9be946e73f8bd090b` (deprecated) | [14470876](ceremony/2021-06-18/keys/14470876) | June 2021 - |
-| Santiago Torres-Arias | `1e1d65ce98b10addad4764febf7dda2d0436b3d3a3893579c0dddaea20e54849` (v9+) `f5312f542c21273d9485a49394386c4575804770667f2ddb59b3bf0669fddd2f` (v5-8) `f40f32044071a9365505da3d1e3be6561f6f22d0e60cf51df783999f6c3429cb` (deprecated) | [15938765](ceremony/2021-06-18/keys/15938765) | June 2021 - |
+#### Online signing
 
-### Emeritus Keyholders
-| Keyholder  | TUF Key ID                                                         | Yubikey Material                                                                                 | Term                  |
-|------------|--------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|-----------------------|
-| Luke Hinds | `bdde902f5ec668179ff5ca0dabf7657109287d690bf97e230c21d65f99155c62` | [14454335](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/14454335) | June 2021 - July 2022 |
+Online signing happens in two situations:
+* A signing event PR has been merged
+* A online (KMS) signature is close to expiry
 
-### Ceremony Overview
-Ceremonies are currently staged in `ceremony/YYYY-MM-DD` according to the date the ceremony was initiated. Ceremonies that resulted in invalid metadata (for whatever reason) and are incomplete are moved into the `ceremony/defunct` directory.
+In practice online signing happens at least every three days because of online signature expiry.
 
-At the end of the ceremony, new repository metadata will be written to a `ceremony/YYYY-MM-DD/repository` directory.
+#### Publishing and automated testing
 
-The ceremony will be completed in five rounds:
+Online signing leads to a "preproduction" deployment at https://sigstore.github.io/root-signing/.
+This is a fully functional TUF repository that is used to run both generic TUF client tests and
+Sigstore specific client tests (with cosign and other sigstore clients). Successful tests lead to
+production deployment at https://tuf-repo-cdn.sigstore.dev/.
 
-![image](https://user-images.githubusercontent.com/5194569/122459506-ffd65e80-cf7e-11eb-8915-e10ac6b50594.png)
+### Workflows
 
-* Round 1: Add Key
-* Round 1.5: Initialize TUF metadata
-* Round 2: Sign Root & Targets
-* Round 3: Sign Snapshot & Timestamp
-* Round 4: Publish final repository.
-
-### Ceremony Instructions
-Before starting the root key ceremony, the community should:
-* Designate the 5 root **keyholders**
```

Comment:
This is something that should likely be re-instated: the idea that the keyholders are important and the community should periodically review the list of keyholders.

So maybe it would make sense to still have a section that lists the keyholders, when they started, and general advice to periodically make sure keyholders are a) trusted by community and b) available for keyholder duties

EDIT: I added a short keyholders section